### PR TITLE
Create theme

### PR DESCRIPTION
**Pull request by anuragpandey1115**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "watch": "tsc --watch",
     "prepublishOnly": "npm run build",
     "test": "jest  -- ./src/__tests__/*.spec.ts",
-    "debug": "node ./bin/fdk login"
```
Comment (vivek8690) [inline package.json]: Don't push these changes @anuragpandey1115

File: package.json [hunk 2]
```diff
@@ -44,6 +44,7 @@
     "@types/url-join": "^4.0.0",
     "@types/validator": "^13.1.3",
     "axios-mock-adapter": "^1.20.0",
+    "copyfiles": "^2.4.1",
```
Comment (vivek8690) [inline package.json]: Remove this dependency as it is not being used @anuragpandey1115

File: template/theme/sections/brands-listing.vue [hunk 3]
```diff
@@ -141,7 +141,7 @@
                       v-show="
                         checkGlide(entry, settings.props.items_per_row.value)
                       "
-                      v-for="(entry, index) in glidePaginate(
+                      v-for="(entry, ) in glidePaginate(
```
Comment (vivek8690) [inline template/theme/sections/brands-listing.vue]: Why is this line changed?

**Review by vivek8690 — COMMENTED** [review]
Add requested changes

File: template/theme/sections/brands-listing.vue (hunk 3 above)
Comment (anuragpandey1115) [reply template/theme/sections/brands-listing.vue]: index use nhi hua tha to error fek rha tha
